### PR TITLE
feat(Textarea): sbanken theme

### PR DESCRIPTION
**Pull request by snorrekim**
## Summary

Sbanken theme of textarea

### CODE REVIEW

File: packages/dnb-eufemia/src/components/textarea/style/themes/dnb-textarea-theme-sbanken.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      --textarea-border-color: var(--sb-color-violet);
+      border-radius: 1.5rem;
+      @include focusRing(null, null, inset);
+    }
+  }
+
+  // disabled
+  &--disabled &__textarea,
+  &__textarea[disabled] {
+    user-select: none; // Safari / Touch fix
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/textarea/style/themes/dnb-textarea-theme-sbanken.scss:64]: Would it possible to "share" such code between the themes? Also other places I have seen interesting code to share: `webkit-line-clamp`.

**Review by tujoworker — COMMENTED** [review]
Nice work 😎 the only question I have, if it would be possible to nicely share some common code between the themes.

File: packages/dnb-eufemia/src/components/textarea/style/themes/dnb-textarea-theme-sbanken.scss (hunk 1 above)
Comment (snorrekim) [reply packages/dnb-eufemia/src/components/textarea/style/themes/dnb-textarea-theme-sbanken.scss:64]: I agree, I'll move all the identical css to the common file.

**Review by tujoworker — APPROVED** [review]

**Pull request merged**